apps/staff/app/page.tsx

```
@@ -703,85 +746,103 @@ function StaffDashboardContent() {
+                                {new Date(announcement.createdAt).toLocaleDateString()}
+                              </span>
+                              <span className="text-border">·</span>
+                              <span className="tabular-nums">
+                                {announcement.recipientCount} scholar
+                                {announcement.recipientCount !== 1 ? 's' : ''}
+                              </span>
+                              {announcement.filters.length > 0 && (
+                                <div className="flex gap-1 ml-1">
+                                  {announcement.filters.map((filter, index) => (
+                                    <Badge
+                                      key={`${filter.type}-${filter.value}-${index}`}
```

Biome (2.4.15)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)